driver-core/src/test/java/com/datastax/driver/core/HostAssert.java

```diff
@@ -141,31 +142,37 @@ public void onDown(Host host) {
         return this;
     }
 
+    @SuppressWarnings("deprecation")
     public HostAssert hasWorkload(String workload) {
         assertThat(actual.getDseWorkload()).isNotNull().isEqualTo(workload);
         return this;
     }
```

Comment:
1) They are only deprecated in OSS driver
2) Not sure if, because some method is deprecated, it should not be tested.